numpy/lib/arraysetops.py

```diff
@@ -627,22 +627,29 @@ def in1d(ar1, ar2, assume_unique=False, invert=False, *, kind=None):
     if ar2.dtype == bool:
         ar2 = ar2 + np.uint8(0)
```

Comment:
Would it make sense to move this this into the table method as well?  I *suspect* it doesn't matter, but if we ever make `np.in1d([True], [1])` an error or warning or whatnot, it might be easier.